index.html

```diff
@@ -24,6 +24,24 @@ <h1>Connect</h1>
       <p>github: vpostyka@github.com</p>
       <p>linkedIn: vlad</p>
     </section>
+    <section>
+      <h2>Leave message</h2>
+      <form name="leave_message">
+        <label for="usersName">Name:</label>
+        <input type="text" name="usersName" required />
+        <br />
```

Comment:
No trailing / needed for html.

Reply:
@toma63 after saving the file it is automatically added.

Reply:
Thought that might be the case.  It's harmless.  May be that the editor is set for xhtml formatting.